src/main/scala/cromwell/engine/backend/runtimeattributes/CromwellRuntimeAttributes.scala

```diff
@@ -39,8 +39,10 @@ object CromwellRuntimeAttributes {
     validatedRuntimeAttributes match {
       case Success(r) => r
       case Failure(f) =>
-        val errorMessages = f.toList.mkString(", ")
-        throw new IllegalArgumentException(s"RuntimeAttribute is not valid: Errors: $errorMessages")
+        throw new IllegalArgumentException() with ThrowableWithErrors {
+          val message = "RuntimeAttribute is not valid."
+          val errors = f
+        }
```

Comment:
Hmm the name of the task is in the logger prefix but not in the message, I can add it there too.
```
2016-01-15 11:59:27,542 cromwell-system-akka.actor.default-dispatcher-8 ERROR - WorkflowActor [UUID(978b16e7):hello]: RuntimeAttribute is not valid.
Missing required keys in runtime configuration for backend 'JES': docker
```